### ISSUE TITLE
Possible incompatibility with jit-grunt

### ISSUE THREAD

**Issue by timkelty** [timkelty/shipit-shared#36]
See @diggy's comment: https://github.com/timkelty/shipit-shared/issues/32#issuecomment-164842377

**Comment by timkelty** [comment]
> https://github.com/timkelty/shipit-shared/issues/32#issuecomment-165407126

@tlartaud so you're using `jit-grunt` and not seeing the same thing @diggy is [here](https://github.com/timkelty/shipit-shared/issues/32#issuecomment-164842377)?

**Comment by tlartaud** [comment]
@timkelty Yeah, but i have only one config parameter. I should try one day to replace jit-grunt with a standard load-grunt-config and see if the problem about orcherstrator binded to late persists.
